File: VB/DSWizardQueryTables/Form1.vb
Imports DevExpress.XtraBars.Ribbon

Namespace DSWizardCustomizationNoSP
    Partial Public Class Form1
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            dashboardDesigner1.DataSourceWizardCustomization = New DataSourceWizardCustomization()
        End Sub
    End Class
End Namespace

File: CS/DSWizardQueryTables/Form1.cs
using DevExpress.XtraBars.Ribbon;

namespace DSWizardCustomizationNoSP
{
    public partial class Form1 : RibbonForm
    {
        public Form1()
        {
            InitializeComponent();
            dashboardDesigner1.DataSourceWizardCustomization = new DataSourceWizardCustomization();
        }
    }
}
